### PR TITLE
Adds Lucy Macartney's profile

### PR DESCRIPTION
**Pull request by lmac-1**
## Changes proposed

Adds Lucy Macartney's profile to LinkFree

## Check List (Check all the applicable boxes)

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1866"></a>

### CODE REVIEW

File: public/data/lmac-1.json [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      "icon": "dollar",
+      "color": "black",
+      "description": "Building a web application with React, NextJS, Prisma and MongoDB"
+    },
+    {
+      "title": "Started working as a technical writer",
+      "date": "April 2022",
+      "icon": "book",
+      "color": "black",
+      "description": ""
```
Comment (Panquesito7) [inline public/data/lmac-1.json:54]: I think you should remove this field if you're not going to use it, IMO.
```suggestion
      "color": "black"
```

Comment (lmac-1) [reply public/data/lmac-1.json:54]: On it! Will make the changes today, thanks!

Comment (lmac-1) [reply public/data/lmac-1.json]: I've removed the unnecessary line. Let me know if anything else is incorrect. Thanks!

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Profile looks good to me, but accidentally you've committed `icons.md`, so please revert the changes in the `icons.md` file! 

PR adding profile should only contain your file!

**Review by EmmaDawsonDev — APPROVED** [review]

**Pull request merged**